internal/pkg/deploy/cloudformation/stack/testdata/workloads/job-test.stack.yml

```diff
@@ -80,6 +80,9 @@ Resources:
             - ContainerPath: /etc/mount1
               ReadOnly: true
               SourceVolume: myEFSVolume
+          DockerLabels:
+            com.amazonaws.ecs.copilot.coollabel: Synecdoche
+            com.amazonaws.ecs.copilot.description: Hello orld!
```

Comment:
How could this `Hello orld` pass the tests 🤔

Reply:
wait wat; seems like the CI hasn't actually run.